src/config.ts

```diff
@@ -316,6 +322,12 @@ export class ConfigurationReader implements vscode.Disposable {
     get rawCTestPath(): string {
         return this.configData.ctestPath;
     }
+    get rawCPackPath(): string {
+        return this.configData.cpackPath;
+    }
+    get cpackVars(): {[key: string]: boolean | number | string | string[] | util.CMakeValue} {
```

Comment:
Are these acutally defined? 

I don't see a setting defined for this in the package.json.

Reply:
In the beginning when things were less clear I think I had in mind something like configure variables (which now I see it doesn't exist as a setting either). I'll remove all traces of cpackVars. The "-D" defines can be sent via cpackArgs.